source/Octopus.Client/OctopusClient.cs

```diff
@@ -447,6 +459,14 @@ OctopusResponse<TResponseResource> DispatchRequest<TResponseResource>(OctopusReq
                 webRequest.Method = "POST";
             }
 
+            var antiforgeryCookie = cookieContainer.GetCookies(cookieOriginUri)
+                .Cast<Cookie>()
+                .SingleOrDefault(c => c.Name.StartsWith(ApiConstants.AntiforgeryTokenCookiePrefix));
+            if (antiforgeryCookie != null)
+            {
+                //webRequest.Headers[ApiConstants.AntiforgeryTokenHttpHeaderName] = antiforgeryCookie.Value;
```

Comment:
should we assigning stuff here like above in async version? Maybe in some common code?